apps/meteor/client/views/omnichannel/directory/CallsContextualBarDirectory.tsx

```diff
@@ -12,16 +12,16 @@ import { VoipInfo } from './calls/contextualBar/VoipInfo';
 import { FormSkeleton } from './components/FormSkeleton';
 
 const CallsContextualBarDirectory = () => {
-	const directoryRoute = useRoute('omnichannel-directory');
+	const { t } = useTranslation();
 
-	const bar = useRouteParameter('bar') || 'info';
 	const id = useRouteParameter('id');
 	const token = useSearchParameter('token');
+	const context = useRouteParameter('context');
```

Comment:
note how a small change in the name of the url parameter caused so many modified files, this could have been done before this pr